osu.Server.Spectator/Program.cs

```diff
@@ -12,6 +15,9 @@ public static class Program
     {
         public static void Main(string[] args)
         {
+            Logger.GameIdentifier = "osu-server-spectator";
+            Logger.Storage = new NativeStorage(Path.Combine(Environment.CurrentDirectory, "Logs"));
```

Comment:
Just to note, `Environment.CurrentDirectory` (along with `IWebHostEnvironment.ContentRootPath`) point to the server project folder, rather than the output folder (`bin/Debug/net5.0`). I guess that's fine? unsure.

Reply:
I think it's fine for now, We can adjust if/when required.